api/v1alpha1/jobset_types.go

```diff
@@ -110,12 +117,32 @@ type Network struct {
 	EnableDNSHostnames *bool `json:"enableDNSHostnames,omitempty"`
 }
 
+// Operator defines the target of a SuccessPolicy or FailurePolicy.
+type Operator string
+
+const (
+	// OperatorAll applies to all jobs in the JobSet.
+	OperatorAll Operator = "All"
+
+	// OperatorAny applies to any job in the Jobset.
+	OperatorAny Operator = "Any"
+)
+
 type FailurePolicy struct {
 	// MaxRestarts defines the limit on the number of JobSet restarts.
 	// A restart is achieved by recreating all active child jobs.
 	MaxRestarts int `json:"maxRestarts,omitempty"`
 }
 
+type SuccessPolicy struct {
+	// Operator determines either All or Any of the selected jobs should succeed to consider the JobSet successful
```

Comment:
Is there a reason why this should be OmitEmpty if you want either All or Any?

Reply:
I guess not - updated.